connector_prestashop/models/account_invoice/importer.py

```diff
@@ -0,0 +1,244 @@
+                  'the PrestaShop Backend.')
+            )
+        return {'journal_id': journal.id}
+
+    def _get_order(self, record):
+        binder = self.binder_for('prestashop.sale.order')
+        return binder.to_internal(record['id_order'])
+
+    @mapping
+    def from_sale_order(self, record):
+        sale_order = self._get_order(record)
+        fiscal_position = None
+        if sale_order.fiscal_position_id:
+            fiscal_position = sale_order.fiscal_position_id.id
+        if sale_order:
+            partner = sale_order.partner_invoice_id
+        else:
+            binder = self.binder_for('prestashop.res.partner')
+            partner = binder.to_internal(record['id_customer'], unwrap=True)
+        return {
+            'origin': sale_order['name'],
+            'fiscal_position_id': fiscal_position,
+            'partner_id': partner.id,
+        }
+
+    @mapping
+    def comment(self, record):
+        return {'comment': _('PrestaShop amount: %s') % record['amount']}
+
+    @mapping
+    @only_create
+    def invoice_lines(self, record):
+        slip_details = record.get(
+            'associations', {}
+        ).get('order_slip_details', []).get(
+            self.backend_record.get_version_ps_key('order_slip_detail'), [])
+        if isinstance(slip_details, dict):
+            slip_details = [slip_details]
+        lines = []
+        order_binding = self._get_order(record)
+        fpos = order_binding.fiscal_position_id
+        shipping_line = self._invoice_line_shipping(record, fpos)
+        if shipping_line:
+            lines.append((0, 0, shipping_line))
+        for slip_detail in slip_details:
+            line = self._invoice_line(slip_detail, fpos)
+            lines.append((0, 0, line))
+        return {'invoice_line_ids': lines}
+
+    def _invoice_line_shipping(self, record, fpos):
+        order_line = self._get_shipping_order_line(record)
+        if not order_line:
+            return None
+        if record['shipping_cost'] == '1':
+            price_unit = order_line['price_unit']
+        else:
+            price_unit = record['shipping_cost_amount']
+        if price_unit in [0.0, '0.00']:
+            return None
+        product = self.env['product.product'].browse(
+            order_line['product_id'][0]
+        )
+        account_id = product.property_account_income_id.id
+        if not account_id:
+            account_id = product.categ_id.property_account_income_categ_id.id
+        if fpos:
+            fpos_obj = self.env['account.fiscal.position']
+            account_id = fpos_obj.map_account(
+                self.session.cr,
+                self.session.uid,
+                fpos,
+                account_id
+            )
+        return {
+            'quantity': 1,
+            'product_id': product.id,
+            'name': order_line['name'],
+            'invoice_line_tax_id': [(6, 0, order_line['tax_id'])],
```

Comment:
```suggestion
            'invoice_line_tax_ids': [(6, 0, order_line['tax_id'])],
```

https://github.com/odoo/odoo/blob/12.0/addons/account/models/account_invoice.py#L1688

I don't know when it was renamed, but it has been a while now. My guess is previous versions have this issue too